modules/sdk-coin-avaxc/src/avaxc.ts

```diff
@@ -62,20 +62,21 @@ import {
   VerifyAvaxcTransactionOptions,
 } from './iface';
 import { AvaxpLib } from '@bitgo/sdk-coin-avaxp';
+import { EthLikeAvaxc } from './EthLikeAvaxc';
 
 export class AvaxC extends BaseCoin {
   static hopTransactionSalt = 'bitgoHopAddressRequestSalt';
-
   protected readonly _staticsCoin: Readonly<StaticsBaseCoin>;
+  private ethLikeAvaxc: any;
```

Comment:
1. Why is composition relationship required?
2. can you name this better EthLikeAvaxc?
just by name, I am not able to understand what is the diff between EthLikeAvaxc and AvaxC.